server/common/api/migrations/0001_initial.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.1.4 on 2019-01-07 01:26
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Class',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('className', models.CharField(max_length=40)),
+                ('minPersonnel', models.IntegerField(default=4)),
+                ('maxPersonnel', models.IntegerField(default=8)),
+                ('price', models.FloatField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Reservation',
+            fields=[
+                ('reesrvationID', models.AutoField(primary_key=True, serialize=False)),
+                ('numOfReserve', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='TimeTable',
+            fields=[
+                ('timeTableIdx', models.AutoField(primary_key=True, serialize=False)),
+                ('date', models.DateField()),
+                ('startTime', models.IntegerField()),
+                ('endTime', models.IntegerField()),
+                ('timezone', models.IntegerField(default=-8)),
+                ('isAvailable', models.BooleanField(default=True)),
```

Comment:
In general, boolean variables should have default value as False. In that regard, this variable is better to be called "isBooked", and set default value to be False.

Reply:
Oh! thank you :) 
Have not ever thought that boolean variables should have default value as False.
It is really really good advice 👍